client/src/index.jsx

```diff
@@ -15,7 +15,14 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    fetch('/api/reviews/40')
+    let location = window.location.toString();
+    location = location.split('/');
+    let businessId = location[location.length - 1];
```

Comment:
I like this method of getting the endpoint. Feels safer than the slicing I'm doing.